### DISCUSSION TITLE
where can I find a .deb?

### DISCUSSION THREAD

**Discussion by RayAndrews56** [hardinfo2/hardinfo2#33] [General]
Is there a debian .deb file?

**Comment by hwspeedy** [comment]
It's still WIP to add the hardinfo2 package to Debian official repositories, so please download a package from prerelease:
https://hardinfo2.org/github/?latest_prerelease

**Reply by RayAndrews56** [reply]
Thanks.  Github seems to enjoy making things hard to find :(  What I'm hoping for is that this version, when you generate a benchmarks report, won't show you results for every CPU ever made -- I really only want the results for my own machine.  I always cut down the report to look like this:

Higher is better:
*CPU CryptoHash	388.27
*CPU Zlib		0.20
*GPU Drawing	3161.88

Lower is better:
*CPU Blowfish	8.69
*CPU Fibonacci	1.62
*CPU N-Queens	30.69
*FPU FFT		3.09
*FPU Raytracing	14.78

**Reply by hwspeedy** [reply]
You cannot compare benchmarks to version 0.x - so be careful with that.

Have you tried starting the app like:
hardinfo2 -n 0

maybe you have not found the project web page as it would have guided you to downloads.
https://hardinfo2.org
But that is for compare - which does not seam to be your purpose.

I'm working on a group feature for benchmark compare - the first part should be done - the second part is the filter on project webpage, which is not done yet. 

You can add a user note to the benchmarks, that you upload with the synchronize button. Start the program like:
hardinfo2 - u RayAndrews56

Later when webpage filter is done - you can compare all of your own machines by filtering by RayAndrews56. The same will work for groups of people like a Single Board Computer community Group in DK: eg. -u SBC-DK

**Reply by RayAndrews56** [reply]
Yeah that's cool, I know all benchmarks are a bit individualistic.  Haven't tried your version yet, I'll report back.

**Reply by hwspeedy** [reply]
Put in a quick hack on the webpage for you to show a group of benchmarks with a specified user note.

example: - filter by Single Board Computers
https://hardinfo2.org?u=SBC

PS: You might have to clear your browser history for the webpage in order for it to work for now.

**Reply by RayAndrews56** [reply]
Cool.

**Reply by RayAndrews56** [reply]
Much nicer report!  Thanks it's worth the upgrade just for that.  Looks like you've put some thought into this.

**Reply by RayAndrews56** [reply]
One little bit of feedback, the:


CPU Blowfish (Single-thread)    <<< why do we need this
----------------------------                 <<< or this

-CPU Blowfish (Single-thread)-   <<< this is sufficient no?  
* This Machine Intel Core i5-6600	4x 3900.00 MHz	3.18

... repetition of the line there seems redundant.  Not to bother you, but I'd format it like this:

-CPU Blowfish (Single-thread)-   3.18
* This Machine Intel Core i5-6600	4x 3900.00 MHz

... so that I could quickly just delete the 'This Machine' lines, since I really don't need to see them ... or maybe just once -- my machine doesn't change between tests.

**Reply by RayAndrews56** [reply]
So again I take the time to format it like this:



-CPU Blowfish (Single-thread)-        3.18
-CPU Blowfish (Multi-thread)-        12.32
-CPU Blowfish (Multi-core)-          12.36
-CPU Zlib-                           12.99
-CPU CryptoHash-                     16.90
-CPU Fibonacci-                     117.05
-CPU N-Queens-                      205.12
-FPU FFT-                            50.87
-FPU Raytracing (Single-thread)-    586.90
-SysBench CPU (Single-thread)-      266.73
-SysBench CPU (Multi-thread)-      1041.68
-SysBench Memory (Single-thread)-  1447.79
-SysBench Memory (Multi-thread)-   2541.26
-GPU Drawing-                       443.08

... or almost like that ... I have the data nicely columnized for fast comparison.

**Reply by RayAndrews56** [reply]
And I see all your benchmarks are 'higher is better' rather than half and half with the standard version.  That's so much better!  Dumb when you 'want' half the benchmarks to report higher numbers and half to report lower numbers.  Just use a mathematical inverse, and get 'higher is better' everywhere.  Bloody good work.

**Reply by RayAndrews56** [reply]
```

-CPU.Blowfish.(Single-thread)-........3.18
-CPU.Blowfish.(Multi-thread)-........12.32
-CPU.Blowfish.(Multi-core)-..........12.36
-CPU.Zlib-...........................12.99
-CPU.CryptoHash-.....................16.90
-CPU.Fibonacci-.....................117.05
-CPU.N-Queens-......................205.12
-FPU.FFT-............................50.87
-FPU.Raytracing.(Single-thread)-....586.90
-SysBench.CPU.(Single-thread)-......266.73
-SysBench.CPU.(Multi-thread)-......1041.68
-SysBench.Memory.(Single-thread)-..1447.79
-SysBench.Memory.(Multi-thread)-...2541.26
-GPU.Drawing-.......................443.08

```
... that's better

**Reply by hwspeedy** [reply]
What you see in CLI/Report is a translated copy of GUI. 
The main purpose is benchmark compare - not numbers for 1 machine (This requires a stable benchmark index)
Numbers for 1 machine is handled by uploading the benchmark numbers to project web server and check on project web server.

I see and understand your idea - maybe in the future...

**Reply by RayAndrews56** [reply]
What I'd do is have the different machines columnized side by side for direct comparison:
```

  
#1 = Ray's Machine: Intel Core i5-6600
#2 = Next machine: .......
#3 = .......

....................................,,.#1...,.#2.....#3

-CPU.Blowfish.(Single-thread)-........3.18....4.0....2.1
-CPU.Blowfish.(Multi-thread)-........12.32...10.0
-CPU.Blowfish.(Multi-core)-..........12.36...12.0
-CPU.Zlib-...........................12.99...13.0
-CPU.CryptoHash-.....................16.90
-CPU.Fibonacci-.....................117.05
-CPU.N-Queens-......................205.12
-FPU.FFT-............................50.87
-FPU.Raytracing.(Single-thread)-....586.90
-SysBench.CPU.(Single-thread)-......266.73
-SysBench.CPU.(Multi-thread)-......1041.68
-SysBench.Memory.(Single-thread)-..1447.79
-SysBench.Memory.(Multi-thread)-...2541.26
-GPU.Drawing-.......................443.08

```

**Reply by RayAndrews56** [reply]
For now, I have some benchmark test to do with various BIOS settings.  This is a 'game'  machine and there's lots of tweeks available in setup.  And I'm going to test them all and put the results in columns as above.
